data/categories.txt

```diff
@@ -785,7 +785,7 @@ shop-car
 en:car shop|U+1F697|U+1F698|U+1F699
 ru:автосалон|магазин
 uk:автосалон|магазин
-de:Auto-Shop|Laden
+de:Autohaus
```

Comment:
Is "Geschäft" better as a generic "shop" or "shops" than "Laden"? What is the difference?